kolibri/core/assets/src/vue/nav-bar/nav-bar-item.vue

```diff
@@ -21,7 +21,11 @@
   module.exports = {
     methods: {
       blur() {
-        this.$refs.link.blur();
+        if (this.$refs.link.$el) {
```

Comment:
could you add some explanation of what's happening with this if-statement? Why is `$el` defined sometimes but not others?